packages/react-template/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "react-template",
+  "version": "0.0.0",
+  "description": "React Template Component",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/you/repo.git"
```

Comment:
Updated.  Once everything is updated with the pull request, do I ```git push``` and then create a new pull request?